src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    void toDraw();
```

Comment:
The method name `toDraw` in the interface `Drawable` is a verb, which is not recommended for method names in interfaces that represent an action. Consider renaming it to simply `draw` to align with the checklist item: 'Don't use verbs for class/interface names'.